src/motion_input_shaping/shaped_axis.py

```diff
@@ -46,6 +66,15 @@ def __init__(self, zaber_axis: Axis, resonant_frequency: float, damping_ratio: f
         # Set the speed limit to the device's current maxspeed so it will never be exceeded
         self.reset_max_speed_limit()
 
+    @property
+    def shaper(self) -> ZeroVibrationShaper | ZeroVibrationStreamGenerator:
```

Comment:
I'd be curious what someone more experienced thinks about this. Returning multiple object types seems like a sketchy practice, but it might be ok in python? Typically this is where you'd use an abstract base class or protocol to define a generic object.

Reply:
I have eliminated this by splitting the class up in the `ShapedAxis` and `ShapedAxisStream`